lib/ISAACConfig.cmake

```diff
@@ -161,34 +152,20 @@ set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} -DBOOST_ALL_NO_LIB)
 
 set(ISAAC_PRIVATE_FOUND true)
 
-################################################################################
-# CUDA LIB
-################################################################################
-if (ISAAC_CUDA)
-    find_package( CUDA 7.0 QUIET)
-    if (!CUDA_FOUND)
-        set(ISAAC_PRIVATE_FOUND false)
-    else()
-        set(ISAAC_INCLUDE_DIRS ${ISAAC_INCLUDE_DIRS} ${CUDA_INCLUDE_DIRS})
-    endif()
-endif()
-
 
 ################################################################################
 # Alpaka LIB
 ################################################################################
-if (ISAAC_ALPAKA)
-    find_package(alpaka QUIET)
-    if (!alpaka_FOUND)
-        set(ISAAC_PRIVATE_FOUND false)
-        set(ISAAC_DEPENDENCY_HINTS ${ISAAC_DEPENDENCY_HINTS} "\n--   Cuda or Alpaka")
-    else()
-        set(ISAAC_INCLUDE_DIRS ${ISAAC_INCLUDE_DIRS} ${alpaka_INCLUDE_DIRS})
-        set(ISAAC_LIBRARIES ${ISAAC_LIBRARIES} ${alpaka_LIBRARIES})
-        set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} ${alpaka_DEFINITIONS})
-        set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} ${ALPAKA_DEV_COMPILE_OPTIONS})
-        set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} "-DISAAC_ALPAKA")
-    endif()
+find_package(alpaka QUIET)
+if (!alpaka_FOUND)
+    set(ISAAC_PRIVATE_FOUND false)
+    set(ISAAC_DEPENDENCY_HINTS ${ISAAC_DEPENDENCY_HINTS} "\n--   Cuda or Alpaka")
+else()
+    set(ISAAC_INCLUDE_DIRS ${ISAAC_INCLUDE_DIRS} ${alpaka_INCLUDE_DIRS})
+    set(ISAAC_LIBRARIES ${ISAAC_LIBRARIES} ${alpaka_LIBRARIES})
+    set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} ${alpaka_DEFINITIONS})
+    set(ISAAC_DEFINITIONS ${ISAAC_DEFINITIONS} ${ALPAKA_DEV_COMPILE_OPTIONS})
```

Comment:
at @psychocoderHPC :
Are these still step by alpaka. 
(@FelixTUD Please correct my question if it is too imprecise.)

Reply:
IMO this is not available anymore. The problem is that ISAAC is not providing a target.
ISAAC needs to provide an `INTERFACE` target and then you can use `target_link_libraries`